### DISCUSSION TITLE
[Feature Request] Consider redaxios instead of axios

### DISCUSSION THREAD

**Discussion by daKmoR** [ing-bank/lion#833] [Ideas] [answered]
https://github.com/developit/redaxios

**Comment by jorenbroekema** [comment]
Interesting, might be worth looking into, have you compared it to Lars' fetch package PR, which is also a wrapper around fetch but with a different API?

**Comment by LarsDenBakker** [comment]
It's similar in functionality and I think filesize. My PR is closer to native fetch API, redaxios is closer to axios API. 

I'm not sure if we could swap axios with redaxios without a breaking change, but if so that would be a major benefit. 

I'm fine with it if you decide to drop my PR and use redaxios, up to you.

**Comment by jorenbroekema** [comment, marked as answer]
Latest ajax package uses Fetch instead of axios/redaxios 🎉
